drivers/usb/usbport/roothub.c

```diff
@@ -190,6 +190,13 @@ USBPORT_RootHubClassCommand(IN PDEVICE_OBJECT FdoDevice,
                 return RHStatus;
             }
 
+            if (Port == 0 ||
+                Port > PdoExtension->RootHubDescriptors->Descriptor.bNumberOfPorts)
```

Comment:
Not `Port >= PdoExtension->RootHubDescriptors->Descriptor.bNumberOfPorts` instead?
Or is `Port` a 1-based value?